languages/go/config.json

```diff
@@ -177,13 +176,13 @@
         "slug": "slices",
         "uuid": "8bc0c6f6-1891-4688-9ed7-a63e3702f655",
         "concepts": ["slices"],
-        "prerequisites": ["basics", "iteration", "numbers"]
+        "prerequisites": ["iteration", "numbers"]
       },
       {
         "slug": "strings",
         "uuid": "b97a271c-4036-4ea8-8c6a-094715a48627",
         "concepts": ["strings"],
-        "prerequisites": ["slices"]
+        "prerequisites": ["basics"]
```

Comment:
I think I have done this already in one of the other merge requests... maybe undo it here?